File: Part4/C++/Function_AssignmentOutOfPlace.cpp
#include <stdio.h>
#include <stdlib.h>

int main()
{
    char userInput[ 100 ];
    
    printf( "Type 1 and press enter.\n" );
    fgets( userInput, sizeof( userInput ), stdin );
    
    // Assignment operator placed incorrectly in declaration
    int enteredInteger atoi = ( userInput );
    
    printf( "The user entered the integer %d\n", enteredInteger );
    
    return 0;
}

File: Part4/Go/Function_AssignmentOutOfPlace.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var userInput string

	fmt.Println( "Type 1 and press enter." )
	fmt.Scanln( &userInput )

	// Assignment operator placed incorrectly in declaration
	enteredInteger strconv.Atoi, _ := ( userInput )

	fmt.Printf( "The user entered the integer %d\n", enteredInteger )
}
